usagov/usagov-2021
# USAGOV-1482: recaptcha v3 upgrade

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1482

## Description

Replace recaptcha v2 with recaptcha v3
## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


To test goto http://localhost/site-issue-report-form and verify the recaptcha shows in the bottom right corner of the page.
<img width="1512" alt="Screenshot 2024-08-05 at 11 40 39 AM" src="https://github.com/user-attachments/assets/bab93bab-4250-4fb2-88dd-bdc8154885bf">

then submit the form without filling out the fields to verify the error validation works.
<img width="1512" alt="Screenshot 2024-08-05 at 1 55 14 PM" src="https://github.com/user-attachments/assets/7149c799-ec05-4648-ba34-4f1db907ca84">

open the dev tools for the page and goto the "Network" tab and fill out the form and submit. Select the webservlet in the 
<img width="1512" alt="Screenshot 2024-08-02 at 1 40 48 PM" src="https://github.com/user-attachments/assets/3e531fcf-599b-4ac0-9efc-e35ca2448371">

Selet the "Form Data" tab and verify the data you submitted is there along with the g-recaptcha-response data
<img width="1512" alt="Screenshot 2024-08-02 at 1 40 34 PM" src="https://github.com/user-attachments/assets/14d82df8-cf39-404a-9f0b-0982a8819062">

Repeat these steps for the Spanish page http://localhost/es/reporte-problemas-en-este-sitio-web

### Change Requirements



### Validation Steps

- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
